internal/operators/ocs/validations.go

```diff
@@ -159,7 +159,7 @@ func (o *operator) canOCSBeDeployed(hosts []*models.Host, nodeInfo *resourceInfo
 	}
 
 	if nodeInfo.numDisks%3 != 0 {
-		status = fmt.Sprint("Total disks on the cluster must be a multiple of 3 of size >= ", minDiskSize, " GB")
```

Comment:
This also does not take into account the non-ha mode

Reply:
OCS won't work for non-HA mode.